lang/java/reef-applications/reef-vortex/pom.xml

```diff
@@ -60,6 +60,11 @@ under the License.
             <artifactId>kryo-serializers</artifactId>
             <version>${kryo-serializers.version}</version>
         </dependency>
+        <dependency>
+        	  <artifactId>asm</artifactId>
+            <groupId>org.ow2.asm</groupId>
+            <version>5.0.3</version>
```

Comment:
Do we depend on this in multiple submodules? If so, we should move the actual version into `../pom.xml` inside of `DependencyManagement`

Reply:
@markusweimer  I have resubmitted a patch following your suggestion. Thank you very much.